uber/templates/preregistration/noise_levels.html

```diff
@@ -1 +0,0 @@
-{% comment %}This section to be overridden only by Magstock, not used by anything else{% endcomment %}
```

Comment:
Since plugins can insert entirely new templates, was there a reason for this dummy template?